firestore.rules

```diff
@@ -0,0 +1,62 @@
+rules_version = '2';
```

Comment:
Again, I'm not clear whether this file is meant to match what we actually plan on deploying, or if it's just a placeholder. We don't plan on requiring auth, and there are a lot of rules here that appear to refer to storage fields that I assume we won't be creating.

Reply:
Added this in .gitignore.

Reply:
Since this file is in gitignore, please remove it from this PR